neo/public/stylesheets/main.css

```diff
@@ -97,3 +99,19 @@ search-bar {
   padding-left: 0;
   padding-right: 0;
 }
+
+#logo_img{
+  z-index = 0;
+  position: absolute;
+  top:2%;
+  left:10%;
+  width : 15%;
+  height: 10%;
+
+}
+
+.btn.btn-info{
```

Comment:
got it. but to make it explicit, could you rename the class to `logo` than the general `btn`?